java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java

```diff
@@ -5393,10 +5395,10 @@ public Object getSubscribedBaseChannel(User loggedInUser, Integer sid) {
      * @xmlrpc.param #param("string", "sessionKey")
      * @xmlrpc.returntype
      *      #array_begin()
-     *          $SystemOverviewSerializer
+     *          $ShortSystemInfoSerializer
      *      #array_end()
      */
-    public List<SystemOverview> listInactiveSystems(User loggedInUser) {
+    public List<ShortSystemInfo> listInactiveSystems(User loggedInUser) {
```

Comment:
We need to make the same changes for `listActiveSystems` methods and it's overloaded variants if there are.